src/sysv/gentoo/sssd-kcm.in

```diff
@@ -6,6 +6,7 @@ description="SSSD Kerberos Cache Manager"
 command="@libexecdir@/sssd/sssd_kcm"
 command_background="true"
 command_args="--logger=files ${SSSD_KCM_OPTIONS}"
+command_user="@SSSD_USER@"
```

Comment:
Shouldn't this be "@SSSD_USER@:@SSSD_USER@"? Or does 'group' default to 'user'?

As a side note, did you consider setting also 'capabilities'?
Do I understand correctly that by default 'bounding set' isn't restricted?

Reply:
Yes, it should be "@SSSD_USER@:@SSSD_USER@"

Gentoo does not compile OpenRC with capabilities support (well, not intentionally: https://bugs.gentoo.org/932090). 

I look at the source of cap_iab and while you can remove capabilities from the binding set, I don't see a way to to see "Remove all capabilities from the binding set, except for these"